src/tools/rbd/action/Group.cc

```diff
@@ -261,6 +261,57 @@ int execute_rename(const po::variables_map &vm,
   return 0;
 }
 
+int execute_info(const po::variables_map &vm,
+                   const std::vector<std::string> &ceph_global_init_args) {
+  size_t arg_index = 0;
+
+  std::string pool_name;
+  std::string namespace_name;
+  std::string group_name;
+
+  int r = utils::get_pool_generic_snapshot_names(
+    vm, at::ARGUMENT_MODIFIER_NONE, &arg_index, at::POOL_NAME, &pool_name,
+    &namespace_name, GROUP_NAME, "group", &group_name, nullptr, true,
+    utils::SNAPSHOT_PRESENCE_NONE, utils::SPEC_VALIDATION_FULL);
+  if (r < 0) {
+    return r;
+  }
+
+  at::Format::Formatter formatter;
+  r = utils::get_formatter(vm, &formatter);
+  if (r < 0) {
+    return r;
+  }
+  Formatter *f = formatter.get();
+
+  librados::Rados rados;
+  librados::IoCtx io_ctx;
+
+  r = utils::init(pool_name, namespace_name, &rados, &io_ctx);
+  if (r < 0) {
+    return r;
+  }
+
+  librbd::RBD rbd;
+  std::string group_id;
+  r = rbd.group_get_id(io_ctx, group_name.c_str(), &group_id);
+  if (r < 0) {
+    return r;
+  }
+
+  if (f) {
+    f->open_array_section("group");
```

Comment:
There is no array in sight here:
```suggestion
    f->open_object_section("group");
```

Reply:
... and let's dump `group_name` too -- it's what `rbd info` command does:
```
    f->dump_string("group_name", group_name);
```
Please post a comment with a cut-and-paste of both plain and formatted output when done.

Reply:
```
  if (f) {
    f->open_object_section("group");
    f->dump_string("group_name", group_name);
    f->dump_string("group_id", group_id);
    f->close_section();
    f->flush(std::cout);
  } else {
    std::cout << "rbd group '" << group_name << "':\n"
              << "\t" << "id: " << group_id << std::endl;
  }
```

Reply:
> Please post a comment with a cut-and-paste of both plain and formatted output when done.

I meant the output of running the new command (i.e. what the user would see), not the code ;)

Reply:
haha, sorry : )
```
root@debian:~/code/c++/ceph/build# ./bin/rbd group info mygroup
rbd group 'mygroup':
        id: 15dc6b8b4567
root@debian:~/code/c++/ceph/build# ./bin/rbd group info mygroup --format json
{"group_name":"mygroup","group_id":"15dc6b8b4567"}
```